shared/test/test_tables.rb

```diff
@@ -141,4 +193,8 @@ def delete_table()
     @tables.delete "/v3/shared-tables/#{@channel_id}/#{@table_name}"
   end
 
+  def populate_table(data, overwrite)
+    @tables.post "/v3/shared-tables/#{@channel_id}?overwrite=#{overwrite}", JSON.generate(data), 'CONTENT_TYPE' => 'application/json;charset=utf-8'
```

Comment:
nit: because the code in AppStorage omits the entire `overwrite` parameter, it might be better to omit it here as well when overwrite != 1